ansible/requirements.yml

```diff
@@ -0,0 +1,3 @@
+- src: https://github.com/redhat-cop/infra-ansible
+  version: master
```

Comment:
would probably want to lock this one on a release?

Reply:
Agreed. Need to merge the PR in the description first, then cut a release

Reply:
@oybed updated